devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -176,11 +186,20 @@ def poll() {
  */
 def refresh() {
 	log.trace "ZigBee DTH - Executing refresh() for device ${device.displayName}"
-	def cmds =
-		zigbee.readAttribute(CLUSTER_DOORLOCK, DOORLOCK_ATTR_LOCKSTATE) +
-		zigbee.readAttribute(CLUSTER_POWER, POWER_ATTR_BATTERY_PERCENTAGE_REMAINING)
+
+	def cmds = []
+	if (device.getDataValue("model") == "E261-KR0B0Z0-HA") {
+		cmds += zigbee.readAttribute(CLUSTER_IAS_ZONE, IAS_ATTR_ZONE_STATUS)
+		cmds += zigbee.readAttribute(CLUSTER_DOORLOCK, DOORLOCK_ATTR_LOCKSTATE)
+		cmds += zigbee.enrollResponse()
```

Comment:
`enrollResponse` shouldn't be needed in `refresh` since it should have been enrolled in the IAS Zone during configuration

Reply:
done